doc/news/README.rst

```diff
@@ -0,0 +1,40 @@
+Recording Changes
+=================
+ 
```

Comment:
There is an empty space. Not a big deal, but if you can get rid of those (:%s/ *$//g in vim), that will save some bits.